src/v/kafka/server/handlers/topics/types.h

```diff
@@ -57,6 +57,10 @@ static constexpr std::string_view topic_property_remote_read
   = "redpanda.remote.read";
 static constexpr std::string_view topic_property_read_replica
   = "redpanda.remote.readreplica";
+static constexpr std::string_view topic_property_retention_local_target_bytes
+  = "retention.local-target.bytes";
```

Comment:
should we keep `.` separator consistent  ?

Reply:
You mean use `retention.local.target.bytes`? I just went for what was in the PRD. I'll ask for Matt's opinion.

Reply:
great, i remember Travis was suggesting that all the properties should be separated with dots. This is also java convention

Reply:
Matt was fine with it. Will update.

Reply:
Done.